pegasus/sites.v3/csedweek.org/views/map.haml

```diff
@@ -111,7 +115,23 @@
     var map_zoom = gmap.getZoom();
     var map_center = gmap.getCenter();
 
-    var map_height = map_width / 2;
+    // Max height of map is 2/3 of screen height so user can always scroll.
+    var max_height = $(window).innerHeight() * .65; 
+
+    var window_aspect_ratio = $(window).innerHeight() / $(window).innerWidth();
+    
+    if  (window_aspect_ratio < 1) {
+      // Landscape window. Use the current 1:2 ratio map size.
+      map_height = map_width / 2;
+
+      // But, make sure this size doesn't exceed our max height.
+       map_height = Math.min(map_height, max_height);
+    } 
+    else {
+      // Portrait window. Just use the max height so that it's taller and more usable.
+      map_height = max_height;
+    }
+
     $('#gmap').height(map_height);
```

Comment:
i generally try to use integers for things like this.. if nothing else it can allow the browser to use a fast rendering path for some parts of the graphics subsystem.